### PR TITLE
feat: Introduce activationThreshold/minMetricValue for Openstack Swift  Scaler

### PR DESCRIPTION
**Pull request by JorTurFer**
Signed-off-by: Jorge Turrado <jorge_turrado@hotmail.es>

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://github.com/kedacore/keda/blob/main/CONTRIBUTING.md#developer-certificate-of-origin-signing-your-work))
- [x] Tests have been added
- [x] A PR is opened to update the documentation on ([repo](https://github.com/kedacore/keda-docs)) *(if applicable)*

Related to https://github.com/kedacore/keda/issues/2800
Related to https://github.com/kedacore/keda-docs/pull/841
Fixes https://github.com/kedacore/keda/issues/3245

### CODE REVIEW

File: tests/scalers_go/openstack_swift/helper.go [hunk 1]
```diff
@@ -0,0 +1,58 @@
+//go:build e2e
```
Comment (zroubalik) [inline tests/scalers_go/openstack_swift/helper.go:1]: I'd name this file openstack_helper, but let's dont block on this.

**Review by zroubalik — APPROVED** [review]

File: tests/scalers_go/openstack_swift/helper.go (hunk 1 above)
Comment (JorTurFer) [reply tests/scalers_go/openstack_swift/helper.go:1]: I had problems with naming in packages, if I name this `openstack_helper`, the linters push me to name the other `openstack_helper_test` and I prefer to no modify the linter rules for this

Comment (zroubalik) [reply tests/scalers_go/openstack_swift/helper.go:1]: 😄 leave it as it it then

**Pull request merged**